Change floor activity to watch instead of streaming

rickstaa opened the pull request:
This PR uses the new UpdateWatchStatus of https://github.com/bwmarrin/discordgo/pull/1291 to set the activity to watching instead of streaming. It follows up on #180.

![image](https://user-images.githubusercontent.com/17570430/204558440-a0686461-2c35-42b3-bf41-ac833c0d0795.png)

> **Warning**
> This PR contains a feature of discordgo that has not yet been released please merge after https://github.com/bwmarrin/discordgo/pull/1291 has been released.

rssnyder commented:
Sweet! Looking forward for that being merged, gonna change all my bots to use this. Please ping me when its merged.

rickstaa commented:
> Sweet! Looking forward for that being merged, gonna change all my bots to use this. Please ping me when its merged.

I will! Happy to collaborate on such an amazing project!🔥

rickstaa commented:
@rssnyder https://github.com/bwmarrin/discordgo/pull/1291 just got merged so we can apply it to all tickers and merge this PR. 🔥

rickstaa commented:
I can fix conflicts tomorrow.

rickstaa commented:
I added the `UpdateWatchStatus` method to the other tickers and rebased the PR onto the master branch. Let's wait till a new version of https://github.com/bwmarrin/discordgo is released.

rickstaa commented:
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

Not stale!

rickstaa commented:
@stephaniehingtgen a new discordgo version was just released so we should be good to go (see https://github.com/bwmarrin/discordgo/releases/tag/v0.27.0). 👍🏻

rssnyder commented:
@rickstaa can you update the dependencies on your branch to use the new release? Then I will validate and merge.

rickstaa commented:
> @rickstaa can you update the dependencies on your branch to use the new release? Then I will validate and merge.

Done 👍.

rssnyder reviewed (approved):
![image](https://user-images.githubusercontent.com/7338312/214614926-1fa6c080-95ab-4ed5-8bd7-b681b074cc24.png)
